CON_PORT: RiCBoard
FILE_NAME: hagai
Battery/name: battery_monitor
Battery/pubHz: 10
Battery/voltageDividerRatio: 5.98
IMU_INIT: 0
GPS_INIT: 0
PPM_INIT: 0
BAT_INIT: 1
DIFF_INIT: 0
DIFF_INIT_OP: 0
DIFF_CLOSE_FOUR: 0
closeLoopNum: 0
switchNum: 0
servoNum: 0
relayNum: 0
URFNum: 0
openLoopNum: 0
